test/error_tracker_test.exs

```diff
@@ -1,8 +1,102 @@
 defmodule ErrorTrackerTest do
```

Comment:
Now that we have Telemetry in main, can we test that events are emitted too?

I saw some references on how to do it on here: https://github.com/dashbitco/broadway/blob/main/test/broadway_test.exs#L311

Reply:
I've added a new test case for telemetry events! 🚀